readme.md

```
@@ -15,120 +15,857 @@
-> Puede ser pública o privada, en cuyo caso deberás compartir los accesos de manera segura. Puedes enviarlos a [alvaro@lidr.co](mailto:alvaro@lidr.co) usando algún servicio como [onetimesecret](https://onetimesecret.com/).
+Marco reutilizable para diseñar e implementar proyectos de **Single Source of Truth (SSOT)** con asistencia de agentes de IA (Cursor, Claude Code, GPT Codex). Actúa simultáneamente como plantilla de proyecto y como cerebro operativo: reglas, agentes, skills y workflows viven bajo `.ai/`. El framework guía el trabajo por fases con gate checks automáticos, trazabilidad completa de decisiones y calidad de datos continua.
 
-### 0.5. URL o archivo comprimido del repositorio
+### **0.4. URL del proyecto:**
 
-> Puedes tenerlo alojado en público o en privado, en cuyo caso deberás compartir los accesos de manera segura. Puedes enviarlos a [alvaro@lidr.co](mailto:alvaro@lidr.co) usando algún servicio como [onetimesecret](https://onetimesecret.com/). También puedes compartir por correo un archivo zip con el contenido
+No aplica
 
+### **0.5. URL o archivo comprimido del repositorio**
 
+Se proporcionara en la entrega final
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)